src/sections/hero.liquid

```diff
@@ -0,0 +1,62 @@
+{% assign settings = section.settings %}
+{% assign img_url = settings.background_image | img_url: 'master' %}
```

Comment:
it would be great if we could use responsive images. if i haven't already, i will install [this library](http://afarkas.github.io/lazysizes/rias/) for this purpose, but it requires using an img element as opposed to a background image. I can abstract the image tag into a snippet later.

Reply:
Ooh thanks for this, I've never seen this library before! I'll change the image to a tag so we can set this up later